otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/akka/http/OverridingRawPatchMatcherImpl.scala

```diff
@@ -0,0 +1,55 @@
+package io.scalac.mesmer.otelextension.instrumentations.akka.http
+
+import akka.http.javadsl.server.Rejected
+import akka.http.scaladsl.server.Directive
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.PathMatcher
+import akka.http.scaladsl.server.PathMatcher.Matched
+import akka.http.scaladsl.server.PathMatcher.Unmatched
+import io.opentelemetry.context.Context
+import io.opentelemetry.instrumentation.api.field.VirtualField
+
+object OverridingRawPatchMatcherImpl {
```

Comment:
it's very difficult to understand what this class does. Do you think you could document that? What exactly is a "raw Path"?